content/articles/boruta-algorithm/index.md

```diff
@@ -0,0 +1,173 @@
+### Getting Started with Boruta Algorithm in Machine Learning 
+Boruta is a Machine Learning algorithm used in feature selection. Feature selection is a process of reducing the number of features in a dataset by identifying features that largely influence the study variable. Feature selection is an important activity in machine learning. For instance, suppose we are required to perform analysis on genetic data. The dataset, in this case, can be tremendously huge, and fitting machine model on them can have significant challenges, which includes:
+
+1. High computaional cost
+2. High processing time
+3. Also, machine learning methods that fit on high dimensional datasets tend to have poor performance than when fit on optimal features of the dataset.
+
+We know that some analysis methods, such as those based on artificial neural networks, work well on large datasets. Due to computational cost, however, it is efficient to provide them with the factors that optimally explain the study variable. Feature selections should not be confused as the process leads to an inferior number of features from a large dataset. This activity is to obtain the optimal features, and this optimal set can also exist in large dimensionality, i.e., 90% of the original dataset.
+
+There are various methods used in feature selection. These methods fall into four broad categories:
+
+1. Filter methods
+
+2. Wrapper methods
+
+3. Embedded methods
+
+4. Hybrid methods
+
+This article will teach how to do feature selection using the Boruta algorithm and implement it in python.
+
+### Prerequisites:
+To get along with this material comfortably, you need to be familiar with:
+1. Python programming skills and familiarity with working on the Jupiter notebook environment
+2. Basic knowledge in [Random forest](https://www.section.io/engineering-education/introduction-to-random-forest-in-machine-learning/)
+
+### Introduction to Boruta Algorithm
+Boruta is a Wrapper method of the *Feature selection* built around the [Random Forest Classifier algorithm](https://www.section.io/engineering-education/introduction-to-random-forest-in-machine-learning/). So how does the Boruta algorithm work?
+
+In the beginning, the algorithm takes features of the original dataset and create a copy of them. On this copy, values in each column are shuffled to attain randomness. These shuffled features are known as **Shandow Features**. The shadow features are then merged with the original features to obtain a new feature space whose dimension is twice the original dataset. The diagram below clarifies the above discussion.
+
+![image](boruta.png)
```

Comment:
Fix all image paths, like this 
`! [Title background] (/engineering-education/collapsing-action-tool-bar-with-android-material-design-in-kotlin/title-bg.png )`